build/cqs.tsv

```diff
@@ -171,41 +172,42 @@ org.eclipse.emf:org.eclipse.emf.common	2.12.0	compile
 org.eclipse.emf:org.eclipse.emf.ecore	2.12.0	compile
 org.eclipse.emf:org.eclipse.emf.ecore.xmi	2.12.0	compile
 org.eclipse.xsd:org.eclipse.xsd	2.12.0	compile
+org.ejml:ejml-core	0.34	compile
+org.ejml:ejml-ddense	0.34	compile
 org.gdal:gdal	2.0.0	compile
-org.geotools:gt-api	18.0	compile
-org.geotools:gt-coverage	18.0	compile
-org.geotools:gt-cql	18.0	compile
-org.geotools:gt-data	18.0	compile
-org.geotools:gt-epsg-hsql	18.0	compile
-org.geotools:gt-geojson	18.0	compile
-org.geotools:gt-geotiff	18.0	compile
-org.geotools:gt-graph	18.0	compile
-org.geotools:gt-grid	18.0	compile
-org.geotools:gt-imageio-ext-gdal	18.0	compile
-org.geotools:gt-main	18.0	compile
-org.geotools:gt-metadata	18.0	compile
-org.geotools:gt-opengis	18.0	compile
-org.geotools:gt-process	18.0	compile
-org.geotools:gt-process-feature	18.0	compile
-org.geotools:gt-referencing	18.0	compile
-org.geotools:gt-render	18.0	compile
-org.geotools:gt-shapefile	18.0	compile
-org.geotools:gt-transform	18.0	compile
-org.geotools:gt-xml	18.0	compile
-org.geotools.ogc:net.opengis.fes	18.0	compile
-org.geotools.ogc:net.opengis.ows	18.0	compile
-org.geotools.ogc:net.opengis.wfs	18.0	compile
-org.geotools.ogc:org.w3.xlink	18.0	compile
-org.geotools.xsd:gt-xsd-core	18.0	compile
-org.geotools.xsd:gt-xsd-fes	18.0	compile
-org.geotools.xsd:gt-xsd-filter	18.0	compile
-org.geotools.xsd:gt-xsd-gml2	18.0	compile
-org.geotools.xsd:gt-xsd-gml3	18.0	compile
-org.geotools.xsd:gt-xsd-ows	18.0	compile
-org.geotools.xsd:gt-xsd-wfs	18.0	compile
+org.geotools:gt-api	20.0	compile
+org.geotools:gt-coverage	20.0	compile
+org.geotools:gt-cql	20.0	compile
+org.geotools:gt-data	20.0	compile
+org.geotools:gt-epsg-hsql	20.0	compile
+org.geotools:gt-geojson	20.0	compile
+org.geotools:gt-geotiff	20.0	compile
+org.geotools:gt-grid	20.0	compile
+org.geotools:gt-imageio-ext-gdal	20.0	compile
+org.geotools:gt-main	20.0	compile
+org.geotools:gt-metadata	20.0	compile
+org.geotools:gt-opengis	20.0	compile
+org.geotools:gt-process	20.0	compile
+org.geotools:gt-process-feature	20.0	compile
+org.geotools:gt-referencing	20.0	compile
+org.geotools:gt-render	20.0	compile
+org.geotools:gt-shapefile	20.0	compile
+org.geotools:gt-transform	20.0	compile
+org.geotools:gt-xml	20.0	compile
+org.geotools.ogc:net.opengis.fes	20.0	compile
+org.geotools.ogc:net.opengis.ows	20.0	compile
+org.geotools.ogc:net.opengis.wfs	20.0	compile
+org.geotools.ogc:org.w3.xlink	20.0	compile
+org.geotools.xsd:gt-xsd-core	20.0	compile
+org.geotools.xsd:gt-xsd-fes	20.0	compile
+org.geotools.xsd:gt-xsd-filter	20.0	compile
+org.geotools.xsd:gt-xsd-gml2	20.0	compile
+org.geotools.xsd:gt-xsd-gml3	20.0	compile
+org.geotools.xsd:gt-xsd-ows	20.0	compile
+org.geotools.xsd:gt-xsd-wfs	20.0	compile
 org.hsqldb:hsqldb	2.3.0	compile
 org.jamon:jamon-runtime	2.4.1	compile
-org.jdom:jdom	1.1.3	compile
+org.jdom:jdom2	2.0.6	compile
```

Comment:
https://dev.eclipse.org/ipzilla/show_bug.cgi?id=18167

@jnh5y do you think it's ok to piggy back on the CQ I did?